sources/packages/forms/src/form-definitions/educationprogramoffering.json

```diff
@@ -3948,409 +2922,896 @@
                       "showWordCount": false,
                       "properties": {},
                       "allowMultipleMasks": false,
-                      "addons": [],
-                      "tree": false,
-                      "lazyLoad": false,
-                      "title": "Panel",
-                      "theme": "default",
-                      "breadcrumb": "default",
-                      "id": "eg5ng19"
+                      "addons": []
                     }
                   ],
                   "placeholder": "",
                   "prefix": "",
-                  "customClass": "",
                   "suffix": "",
                   "multiple": false,
-                  "defaultValue": null,
-                  "protected": false,
                   "unique": false,
-                  "persistent": true,
+                  "refreshOn": "",
+                  "dataGridLabel": false,
+                  "widget": null,
+                  "showCharCount": false,
+                  "showWordCount": false,
+                  "allowMultipleMasks": false,
+                  "addons": [],
+                  "tree": true,
+                  "lazyLoad": false,
+                  "id": "efo4roq"
+                },
+                {
+                  "label": "BreaksAndWeeks",

```

Comment:
do we still need this?

Reply:
Nope, it is a left over, I will remove it.

Reply:
Removed from the form definition.